src/zino/tasks/linkstatetask.py

```diff
@@ -51,6 +57,20 @@ async def run(self):
 
         self._update_interfaces(attrs)
 
+    async def poll_single_interface(self, ifindex: int):
+        """Polls and updates a single interface"""
+        snmp = SNMP(self.device)
+        poll_list = [("IF-MIB", column, str(ifindex - 1)) for column in BASE_POLL_LIST]
+        result = await snmp.getnext2(*poll_list)
```

Comment:
would it not be better to use a get operation instead of a getnext? I assume getnext2 is used here since the current  get function does not support multiple values. I think it would be better to either accept the slight inefficiency and do multiple get requests, or make a get2. Correct me if im wrong on my assumptions.

Reply:
Actually, it's re-implemented precisely in the same way Zino 1 does it - and yes, one of the reasons for that is that we get multiple values in a single request.

Go right ahead and make a `get2`, if you'd like, but it's out of scope for this PR.  Switching to individual get requests would require 6 pairs of request/reply packets rather than one pair, and these things begin to add up when there are a lot of routers in the mix.